lib/sanity.ts

```diff
@@ -145,7 +145,7 @@ export const getBlogs: (limit: number) => Promise<SanityBlog[]> = async (
 
 export const getAllBlogs: () => Promise<SanityBlog[]> = async () => {
   const allBlogs: SanityBlog[] = await client.fetch(
-    `*[_type == 'blog' && !(_id in path('drafts.**'))]  {
+    `*[_type == 'blog' && !(_id in path('drafts.**'))] | order(_createdAt desc)  {
```

Comment:
What is the context for adding this here? Checking the sanity documentation to educate myself and seeing a different pattern. 

https://www.sanity.io/docs/sort-orders

Reply:
@bdougie the doc link you looked at describes how you can display sorted documents in the **Sanity Studio**. But here the `order(_createdAt desc)` is used to **query** the sorted data as an API response.

Please take a look at this URL
https://www.sanity.io/docs/query-cheat-sheet#b5aec96cf56c

Reply:
Excellent, thanks for the clarification